### PR TITLE
Hfb 1549 create attachment file endpoint

### PR DESCRIPTION
**Pull request by ivan-nezhura**

### CODE REVIEW

File: src/Models/Document/AddDocumentAttachmentFile.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+     * @param string $type
+     * @param string $name
+     * @param string $file
+     * @param array $meta
+     */
+    public function addDocumentAttachmentFile(
+        string $category,
+        string $type,
+        string $name,
+        string $file,
```
Comment (StAngerGordun) [inline src/Models/Document/AddDocumentAttachmentFile.php:22]: `string $fileContent` will be better

File: src/Services/DocumentsService.php [hunk 2]
```diff
@@ -428,6 +429,27 @@ public function addDocumentAttachments(
...
+     * @param string $documentId
+     * @param AddDocumentAttachmentFile $addDocumentAttachmentFile
+     * @return DocumentAttachment
+     * @throws Exception
+     */
+    public function addDocumentAttachmentFile(
+        string $documentId,
+        AddDocumentAttachmentFile $addDocumentAttachmentFile
+    ): DocumentAttachment {
+        $url = $this->resolveEndpoint("/documents/$documentId/document-attachments/files");
```
Comment (StAngerGordun) [inline src/Services/DocumentsService.php:442]: `$url = $this->resolveEndpoint("/documents/{$documentId}/document-attachments/files");`

**Review by StAngerGordun — APPROVED** [review]

File: src/Models/Document/AddDocumentAttachmentFile.php (hunk 1 above)
Comment (ivan-nezhura) [reply src/Models/Document/AddDocumentAttachmentFile.php]: fixed

File: src/Services/DocumentsService.php (hunk 2 above)
Comment (ivan-nezhura) [reply src/Services/DocumentsService.php]: fixed

**Review by StAngerGordun — DISMISSED** [review]

**Review by AlexanderTereshchenko — APPROVED** [review]

**Pull request merged**